test/reachsets.jl

```diff
@@ -101,5 +101,36 @@ end
     @test tspan(c) == tspan(d) == 0..1
     @test isequivalent(set(overapproximate(c, Zonotope)), set(R))
     @test isequivalent(set(overapproximate(d, Zonotope)), set(R))
+                    
+end
 
+@testset "Taylor model reach-sets with non-float coefficients" begin
+    
+    using TaylorModels:Taylor1
+    
+    order = 4
+    X0 = BallInf(ones(2), 0.1)
+    t = Taylor1(order+1)
+    
+    A = [0 0..0.1; 0..0.2 0..0.3]*t
+    n = size(A, 1)
+    R = ReachSet(X0, 0 .. 1)
+    RTM = overapproximate(R, TaylorModelReachSet)
+    X0_tm = set(RTM)
+    C = Vector{Vector{TaylorN{IntervalArithmetic.Interval{Float64}}}}(undef, n)
+    Y = [X.pol.coeffs[1] for X in X0_tm]
+    for i in 1:n
```

Comment:
since `n = 2` here i think it'd be more clear if we just write down the defining equations of `C` instead of loops -- if you prefer the general form, then i suggest to write a helper function